window_manager/include/core/XWWindow.hpp

```diff
@@ -4,11 +4,19 @@
 #include <memory>
 #include <future>
 
+#include <boost/log/trivial.hpp>
+
 #include <data_types/system/WindowConfig.hpp>
 #include <data_types/rendering/RenderingContext.hpp>
 
 #include <X11/Xlib.h>
 
+namespace rcbe::rendering {
```

Comment:
Good as of now, but for future, consider to relocate all these into special forwarding header.

Reply:
Okay, will keep that in mind